ajax/libs/material-kit/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "material-kit",
+  "version": "1.1.0",
+  "description": "A Badass Bootstrap UI Kit based on Material Design",
+  "filename": "js/material-kit.min.js",
+  "homepage": "https://www.creative-tim.com/product/material-kit",
+  "author": "creative-tim",
+  "keywords": [
+    "material",
+    "bootstrap",
+    "design",
+    "google",
+    "twitter"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https+git://github.com/creativetimofficial/material-kit.git"
```

Comment:
@extend1994 @sufuf3  Why do we need the `git+https://github.com/creativetimofficial/material-kit.git?`. The URL  shouldn't it be just "https://github.com/creativetimofficial/material-kit.git"? In the other libraries that were added was like this..?